runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -181,11 +183,16 @@ public void onScriptEvent(ScriptCallbackEvent event)
 				}
 
 				boolean tagSearch = searchInput.startsWith(TAG_SEARCH);
+				boolean tagSearchAlt = searchInput.startsWith(TAG_SEARCH_ALT);
```

Comment:
This would probably be better implemented by changing `TAG_SEARCH` into a Pattern (`^(tag:|#)`) and performing the substring operation based on the length of the match. This way we don't introduce these extra variables/if-branches to perform the exact same behavior again.

Reply:
I would rather do one or another, but we had this discussion before, I dont think we came to any reasonable conclusion

Reply:
# is easier/faster to type, I'd be in favour of that if we only have one

Reply:
changed it to use regex